### ISSUE TITLE
[Accessibility] Unable to read search box when results combo box is displayed

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#11643]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.73.1 (and all previous ones)
- OS Version: Windows 11 and 10, both desktop and web version

Steps to Reproduce:

1. Open VS Code while running any screen reader (I use JAWS)
2. Open any popup which displays a set of results based on a search box (e.g. Ctrl+Shift+P for Command Palette, Ctrl+P to open file, or click in language mode at status bar to change language)
3. Type some text to see results
4. Try to move left and right to read the text box, or backspace to delete something
5. Try to tab to move back to the search box

When any result is shown, it's impossible to edit the search term for a blind person, because the combo box steals the focus. I'd like to edit the text, read by letter and word, delete...
Maybe a good fix would be allowing tab to move focus from search box to results list and vice-versa in a circular way.

Just to be clear: editing is possible, the cursor moves left and right, I can select text, delete, whatever I want. The problem is the feedback for a blind person using an screen reader.
